doc/api/repl.markdown

```diff
@@ -226,10 +229,13 @@ There are a few special REPL commands:
     >.save ./file/to/save.js
   - `.load` - Load a file into the current REPL session.
     >.load ./file/to/load.js
+  - `.scope` - Show both global and local(scope) variables
```

Comment:
Got it, I just did introduce this way to see what's your opinion, and now i will revert it, hold on